test/gtest/transforms.cpp

```diff
@@ -43,4 +44,38 @@ TEST_F(P4C_IR, Transform) {
     EXPECT_EQ(e, n);
 }
 
+TEST_F(P4C_IR, InlineBlock) {
+    struct TestTrans : public Transform {
+        explicit TestTrans(const IR::Constant *c) : c(c) {}
+
+        const IR::Node *postorder(IR::AssignmentStatement *a) override {
+            a->right = c;
+            return IR::inlineBlock(*this, {a, a});
+        }
+
+        const IR::Constant *c;
+    };
+
+    const auto *c2 = new IR::Constant(2);
+    const auto *c4 = new IR::Constant(4);
+    const auto *pe = new IR::PathExpression(new IR::Path("foo"));
+    const auto *stmt = new IR::BlockStatement(IR::IndexedVector<IR::StatOrDecl>(
+        {new IR::AssignmentStatement(pe, c2),
```

Comment:
You can get nicer formatting by using a trailing comma.

https://lists.llvm.org/pipermail/cfe-commits/Week-of-Mon-20140317/101748.html

Reply:
Interesting, I did not even know that the trailing comma is accepted there.